PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -487,9 +516,24 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     docString = cms.string("MC matching to status==1 electrons or photons"),
 )
 
+electronMCTableNew = cms.EDProducer("CandMCMatchTableProducer",
+    src     = electronTable.src,
+    mcMapDressedLep = cms.InputTag("electronsMCMatchForTableAlt"),
+    mcMap   = cms.InputTag("electronsMCMatchForTable"),
+    mapTauAnc = cms.InputTag("matchingElecPhoton:hasTauAnc"),
+    objName = electronTable.name,
+    objType = cms.string("ElectronDressed"), 
+    branchName = cms.string("genPart"),
+    docString = cms.string("MC matching to status==1 electrons or photons"),
+    genparticles     = cms.InputTag("finalGenParticles"), 
+)
+
```

Comment:
this "electronMCTableNew" is not used anymore in any sequence.
we should add modification into electronMCTable and have a simple way to take or not the new settings.